scripts/download-deployments.sh

```diff
@@ -18,5 +18,5 @@ echo "downloading 'multus-daemonset.yaml'..."
 curl -sS https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset.yml \
     -o deploy/multus-daemonset.yaml
 echo "downloading 'neonvm.yaml'..."
-curl -sSL https://github.com/neondatabase/neonvm/releases/download/v0.4.5/neonvm.yaml \
+curl -sSL https://github.com/neondatabase/neonvm/releases/download/v0.4.6/neonvm.yaml \
```

Comment:
is there a way we can get this from the go.mod to avoid hard coding it here?

Reply:
```bash
$ go list -m -f '{{ .Version }}' github.com/neondatabase/neonvm
v0.4.5
```

Reply:
Opened #62 to resolve this after merging.